MODULE.bazel

```diff
@@ -1,6 +1,137 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_twirl")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "buildifier_prebuilt", version = "7.3.1")
```

Comment:
I think we can mark this as `dev_dependency`

Reply:
See my comment here:
https://github.com/lucidsoftware/rules_scala/pull/74#discussion_r1873510258